### PR TITLE
Magento2 Theme Snippets

### PR DESCRIPTION
**Pull request by parthviroja**
Magento2 Theme Snippets

### CODE REVIEW

File: repository/m.json [hunk 1]
```diff
@@ -66,6 +66,18 @@
...
 		},
 		{
+			"name": "Magento2 Snippets",
+			"details": "https://github.com/parthviroja/sublime-magento2-snippets",
+			"labels": ["mage", "magento", "magento2", "snippets", "theme", "module"],
+			"author" : "Parth M. Viroja",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```
Comment (wbond) [inline repository/m.json]: This needs to be changed to `"tags": true` and then you need to create a semver tag on the repo (e.g. 1.0.0).

**Comment by wbond** [comment]
The file https://github.com/parthviroja/sublime-magento2-snippets/blob/master/packages.json can be removed from your repo.

**Comment by wbond** [comment]
Also, you should expand your readme to explain what type of snippets you are providing - perhaps a list?

**Comment by parthviroja** [comment]
Thank @wbond to suggest me.

**Comment by FichteFoll** [comment]
You can append commits to existing pull requests by pushing onto the branch you opened it from. Just FYI.

**Pull request closed without merging**